test/functional/local_transaction_controller_test.rb

```diff
@@ -342,13 +314,8 @@ def subscribe_logstasher_to_postcode_error_notification
       assert response.body.include?("Search")
     end
 
-    should "expose the 'missing interaction' error to the view" do
-      location_error = assigns(:location_error)
-      assert_equal "laMatchNoLink", location_error.postcode_error
-    end
-
-    should "log the 'missing interaction' error to the view" do
-      assert_equal(LogStasher.store["postcode_error_notification"], postcode_error: "laMatchNoLink")
+    should "expose the 'missing interaction' error in the view" do
+      assert_match "laMatchNoLink", response.body
```

Comment:
I think this should be:

```
assert_match "We do not know if they offer this service.", response.body
```

At the moment the test passes as `response.body` contains `laMatchNoLink` as its part of the analytics `data-track-action` but it looks like it should be testing for the error message. However `laMatchNoLink` has been removed from `location_error.rb` so its a bit confusing to see where that comes from.